src/components/Comments.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getComments } from '../utils/api';
+// import Article from './Article';
+
+const Comments = () => {
+	const { article_id } = useParams();
+	const [comments, setComments] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
+
+	useEffect(() => {
+		getComments(article_id).then((fetchedComments) => {
+			setIsLoading(false);
+			setComments(fetchedComments);
+		});
+	}, [article_id]);
```

Comment:
we want to make isLoading true at the very start of this useEffect

Reply:
`const Comments = () => {
	const { article_id } = useParams();
	const [comments, setComments] = useState([]);
	const [isLoading, setIsLoading] = useState(true);

	useEffect(() => {
		getComments(article_id).then((fetchedComments) => {
			setIsLoading(true);
			setComments(fetchedComments);
		});
	}, [article_id]);`

I thought It was set in the line 9: `const [isLoading, setIsLoading] = useState(true);`